docs/zh-cn/1.3.0/user_doc/cluster-deployment.md

```diff
@@ -261,35 +243,53 @@ mysql -uroot -p
     # TLS协议的邮箱设置为true，否则设置为false
     starttlsEnable="true"
 
-    # 邮件服务地址值，参考上面 mailServerHost
-    sslTrust="smtp.exmail.qq.com"
-
     # 开启SSL协议的邮箱配置为true，否则为false。注意: starttlsEnable和sslEnable不能同时为true
     sslEnable="false"
 
-    # excel下载路径
-    xlsFilePath="/tmp/xls"
+    # 邮件服务地址值，参考上面 mailServerHost
+    sslTrust="smtp.qq.com"
+
 
     # 业务用到的比如sql等资源文件上传到哪里，可以设置：HDFS,S3,NONE，单机如果想使用本地文件系统，请配置为HDFS，因为HDFS支持本地文件系统；如果不需要资源上传功能请选择NONE。强调一点：使用本地文件系统不需要部署hadoop
     resourceStorageType="HDFS"
 
-    # 资源上传根路径,主持HDFS和S3,由于hdfs支持本地文件系统，需要确保本地文件夹存在且有读写权限
-        resourceUploadPath="/data/dolphinscheduler"
-
-    # 具备权限创建resourceUploadPath的用户
-        hdfsRootUser="hdfs"
-
     # 这里以保存到本地文件系统为例
-    #注：但是如果你想上传到HDFS的话，NameNode启用了HA，则需要将core-site.xml和hdfs-site.xml放到conf目录下，本例即是放到/opt/dolphinscheduler/conf下面，并配置namenode cluster名称；如果NameNode不是HA,则修改为具体的ip或者主机名即可
+    #注：但是如果你想上传到HDFS的话，NameNode启用了HA，则需要将hadoop的配置文件core-site.xml和hdfs-site.xml放到conf目录下，本例即是放到/opt/dolphinscheduler/conf下面，并配置namenode cluster名称；如果NameNode不是HA,则修改为具体的ip或者主机名即可
     defaultFS="file:///data/dolphinscheduler"    #hdfs://{具体的ip/主机名}:8020
 
-
     # 如果ResourceManager是HA，则配置为ResourceManager节点的主备ip或者hostname,比如"192.168.xx.xx,192.168.xx.xx"，否则如果是单ResourceManager或者根本没用到yarn,请配置yarnHaIps=""即可，我这里没用到yarn，配置为""
     yarnHaIps=""
 
     # 如果是单ResourceManager，则配置为ResourceManager节点ip或主机名，否则保持默认值即可。我这里没用到yarn，保持默认
     singleYarnIp="ark1"
```

Comment:
# if not use hadoop resourcemanager, please keep default value; if resourcemanager HA enable, please type the HA ips ; if resourcemanager is single, make this value empty
yarnHaIps="192.168.xx.xx,192.168.xx.xx"

# if resourcemanager HA enable or not use resourcemanager, please skip this value setting; If resourcemanager is single, you only need to replace yarnIp1 to actual resourcemanager hostname.
singleYarnIp="yarnIp1"